docs/comparison.rst

```diff
@@ -0,0 +1,83 @@
+
+Comparison with other simulators
+================================
+
+pyuvsim is designed to calculate visibilities from point sources and a given primary beam model. It does this through direct calculation of the full-polarization Radio Interferometer Measurement Equation (RIME), fully parallelized across a set of MPI processes. Its development has progressed with the goal of providing a tool that can be used reliably and flexibly, but most of all it has focused on avoiding effects that confound foreground modeling efforts in 21cm cosmology.
+
+As such, it has relatively poor performance compared with several other simulators. Deciding which tool to use will ultimately depend on what factors are important to the user. We recommend pyuvsim as a reference tool to verify the results of other simulators, and for producing limited datasets for comparison with observations (particularly on long baselines). Users will need access to a lot of memory and a large number of processor cores to take advantage of pyuvsim's accuracy and precision, ideally on a dedicated high-performance computing cluster.
+
+On this page, we compare pyuvsim's features and performance to several other visibility simulators.
+
+Precision Radio Interferometry Simulator (PRISim)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A python package for generating full-sky simulations.
+
+Advantages:
+
+1. Faster due to vectorized calculations. 
+2. Well-tested through years of use.
+3. Support for a range of sky models including point sources, diffuse emission, and spectral cubes.
+4. Support for tracking as well as transit telescopes.
+5. Parallelized over baselines, frequencies, or sky model/catalog.
+6. Has been tested using imaging with CASA.
+
+Disadvantages:
+
+1. MPI overhead is large if time axis in simulation is long.
+2. Supports only one polarization at a time.
```

Comment:
Also can only support real power beams, so not cross-polarizations (e.g. PRISim supports 'XX', 'YY' but not 'XY', 'YX')

Reply:
I think this means it cannot do accurately simulate polarized sources.

Reply:
I'd state it as "Limited to accurately simulating the scalar (total intensity or diagonal components of) radio interferometer measurement equation if either the polarized sky emission and/or the off-diagonal terms in beam pattern are/is negligible." Or something along these lines. I think that's a fair and objective statement.

Reply:
Ok. I do think that clarifying that 'XX' and 'YY' are supported but 'XY' and 'YX' aren't in some sort of parenthetical would help me understand what your sentence means.

Reply:
Let me try a longer explanation. What I am referring to is the standard leakage from polarized emission to total intensity through off-diagonal terms in the beam and a polarized sky, through a matrix product of the Mueller beam and the Stokes Intensity. For this leakage to be significant, significant polarized emission and significant off-diagonal beam terms are both essential. If either one of them is negligible, then the leakage is negligible too. I was just stating that PRISim works accurately for limited applications where the leakage is negligible, in other words when the matrix product can be effectively scalarized because the product of off-diagonal beam times polarized emission is negligible.

Reply:
I just took a crack at re-writing this, does the following work for you @nithyanandan?

Limited to accurately simulating the total intensity (stokes I) radio interferometer measurement equation, which is accurate if the circularly polarized sky emission is negligible and
either the linearly polarized sky emission is negligible or the off-diagonal terms
in beam pattern (e.g. XY or YX for linear polarization) are negligible.

Reply:
Actually that's not correct either. If there's true stokes U on the sky, you can't see it with any combination of 'XX' and 'YY'. The only polarized emission you can see with those is Stokes Q.

Reply:
So my version reads:

Limited to accurately simulating the total intensity (stokes I) radio interferometer measurement equation,
which is accurate the polarized sky emission is negligible and the off-diagonal terms
in beam pattern (e.g. XY or YX for linear polarization) are negligible.

Reply:
Sounds good.